.github/workflows/shared-build-wasm.yml

```diff
@@ -89,5 +89,6 @@ jobs:
             bindings/wasm/node
             bindings/wasm/web
             bindings/wasm/examples/dist
```

Comment:
This can be removed, no?
```suggestion
```

Reply:
I don't think so, that artifact is used for the cypress e2e test